content/Mobile_Dev/WebAuthn/index.adoc

```diff
@@ -0,0 +1,22 @@
+== WebAuthn development for mobile devices
+At this stage you have selected WebAuthn as your primary MFA option for your mobile solution. Through the rest of this series we are going to cover a variety of different steps and concepts that will help you along your implementation journey. Don’t be frightened if you are not familiar with WebAuthn, we are going to begin this section by giving an overview on what WebAuthn is, along with more resources for you to take a deeper dive into the concepts, and a how-to on deploying a sample application. Once you feel ready then proceed to the following steps where we will dive into OS and ecosystem specific concepts. 
```

Comment:
* Consider rephrasing the content in this paragraph to more directly talk about webauthn development on mobile concepts / learning objectives and remove statements like 'Don't be frightened'.
* Consider moving the OS specific guide towards the top of the page and put the questions below it.

Reply:
Removed some of the fluff in the paragraph, let me know your thoughts